src/node/heartbeat/sync_mempool.rs

```diff
@@ -31,20 +31,66 @@ pub async fn sync_mempool<B: Blockchain>(
             .into_iter()
             .map(|(_, r)| (r.chain_sourced, r.mpn_sourced))
             .collect::<Vec<_>>();
+        let mut fee_skipped_txs = 0;
+        let mut unknown_skipped_txs = 0;
         for (chained_source_txs, mpn_sourced_txs) in resps {
             for tx in chained_source_txs {
```

Comment:
This only filters those transactions that are added in the sync_mempool procedure. How about transactions added through /transact api function?